src/riscv_private.h

```diff
@@ -59,8 +61,12 @@ typedef struct block {
     uint32_t insn_capacity;    /**< maximum of instructions encompased */
     struct block *predict;     /**< block prediction */
     rv_insn_t *ir;             /**< IR as memory blocks */
+#if RV32_HAS(JIT)
+    bool hot; /**< Determine the block is hotspot or not */
+#endif
 } block_t;
 
+#if !RV32_HAS(JIT)
```

Comment:
Drop this preprocessor directive since it is a well-defined declaration. Keeping it doesn't hurt.

Reply:
Add comments to annotate the functionality of this struct.